### PR TITLE
Prep for deploying to dev vSphere

### PR DESCRIPTION
**Pull request by bonniegee**
#73 is basically fixed with what's already in this branch--I think it's ready to be merged with `base`.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -27,3 +27,5 @@
 /lib/seeds/*
 db/data.yml
 
+# Ignore environment variable file
+.env
```
Comment (helrond) [inline .gitignore:31]: Is this intended to ignore `.env.dev` or just `.env`?

File: README.md [hunk 2]
```diff
@@ -44,7 +46,17 @@ The application will be available in your browser at `http://localhost:3000`.
...
 Because the application makes HTTP using Javascript, CORS needs to be implemented on your ArchivesSpace instance. See [as-cors](https://github.com/RockefellerArchiveCenter/as-cors) for an example of how to do this with an ArchivesSpace plugin.
 
-## Installation
+## Deploying in production
+
+In order to deploy in production, `dm_log` expects the following environment variables to be set (the values associated with each of these may change):
+
+```
+MYSQL_ROOT_PASSWORD=example
+SECRET_KEY_BASE=Wa4Kdu6hMt3tYKm4jb9p4vZUuc7jBVFw
```
Comment (helrond) [inline README.md:55]: Might be worth adding a note as to why this is not needed when using `docker-compose.yml`?

**Review by helrond — COMMENTED** [review]
Looks good - a couple of questions mostly for my own enlightenment...

File: .gitignore (hunk 1 above)
Comment (bonniegee) [reply .gitignore:31]: I'm pretty sure I just copied the approach in cartographer: https://github.com/RockefellerArchiveCenter/cartographer

**Review by helrond — APPROVED** [review]

**Pull request merged**